tests/conftest.py

```diff
@@ -1,6 +1,20 @@
+from snuba import settings
+from snuba.clickhouse.native import ClickhousePool
 from snuba.environment import setup_sentry
 
 
-def pytest_configure():
-    """ Set up the Sentry SDK to avoid errors hidden by configuration """
+def pytest_configure() -> None:
+    """
+    Set up the Sentry SDK to avoid errors hidden by configuration.
+    Ensure the snuba_test database exists
+    """
     setup_sentry()
+
+    # There is only one cluster in test, so fetch the host from there.
+    cluster = settings.CLUSTERS[0]
+
+    connection = ClickhousePool(
+        cluster["host"], cluster["port"], "default", "", "default",
+    )
+
+    connection.execute("CREATE DATABASE IF NOT EXISTS snuba_test;")
```

Comment:
shouldn't you pick the name from the cluster i the settings ? I don't think there is the need to hardcode it in two places.

Also, as we drop the tables at the end we should probably drop the testingn DB as well.

Reply:
Updated the database name. We don't currently drop the tables at the end. Would like to tackle that part separately.

Reply:
ok sure